awx/main/management/commands/callback_stats.py

```diff
@@ -23,7 +23,7 @@ def handle(self, *args, **options):
                         ('last hour:     ', '1 hour'),
                     ):
                         cursor.execute(
-                            f"SELECT MAX(id) - MIN(id) FROM {relation} WHERE modified > now() - '{interval}'::interval;"
+                            f"SELECT MAX(id) - MIN(id) FROM {relation} WHERE created > now() - '4 hours'::interval AND modified > now() - '{interval}'::interval;"
```

Comment:
So technically speaking, we've *got* to set an upper bound on this _somehow_.

So if somebody's events are taking *more* than 4 hours to be emitted and saved, they won't show up in these stats.  But if that's you - if you have _no_ stdout, and this command consistently shows "0" for job events, we know you're *very* drastically behind and are probably inserting at a *painfully* slow rate.